### PR TITLE
    fix: Pinning KinD version to 0.7.0 for OS3 tests

### PR DESCRIPTION
**Pull request by agatakrajewska**
- [x] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

Pinning KinD binary version to 0.7.0, compatible with Kubernetes v1.11. We're using that specific version to mimic an OS3 cluster in our integration tests.

### CODE REVIEW

File: test/setup/platforms/kind.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ const clusterName = 'kind';
 
 export async function setupTester(): Promise<void> {
   const osDistro = platform();
-  await download(osDistro);
+  await download(osDistro, 'v0.8.1');
```
Comment (shaimendel) [inline test/setup/platforms/kind.ts:10]: why did you remove the version here? this parameter looks like mandatory in `download`

File: test/setup/index.ts [hunk 2]
```diff
@@ -133,7 +133,7 @@ export async function deployMonitor(): Promise<string> {
     await predeploy(integrationId);
 
     // TODO: hack, rewrite this
-    const imagePullPolicy = testPlatform === 'kind' ? 'Never' : 'Always';
+    const imagePullPolicy = testPlatform === 'kind' || testPlatform === 'openshift3' ? 'Never' : 'Always';
```
Comment (agranado2k) [inline test/setup/index.ts:136]: A small comment. Just to avoid the code gets complicated. What do you think if we create a function, per platform, that specify which policy we should use to image pull?
Not sure if it will help us, or make the code harder to understand.

File: test/setup/platforms/kind.ts (hunk 1 above)
Comment (agatakrajewska) [reply test/setup/platforms/kind.ts:10]: No no, added a version here!

File: test/setup/index.ts (hunk 2 above)
Comment (agatakrajewska) [reply test/setup/index.ts:136]: I think at this point I'd leave it here as I it would require extending the whole interface so we can do that once it does get complicated, wdyt?
